ceph/rados/core_workflows.py

```diff
@@ -1123,3 +1123,34 @@ def get_cephdf_stats(self, pool_name: str = None) -> dict:
                 return {}
 
         return cephdf_stats
+
+    def get_pg_state(self, pg_id):
```

Comment:
This piece of code already exists in master https://github.com/red-hat-storage/cephci/blob/fb69c0135e36a7a822c5092aef256825f489dfca/ceph/rados/core_workflows.py#L1127 , 

and was checked-in as part of this : https://github.com/red-hat-storage/cephci/pull/2308/files#diff-816d963eb5fd64b2936270bd9b278e3709383f5d4c3de6851ca5814603232069R1127 .

Please rebase the code and check.

Reply:
I rebase the code in my local machine and copied the code to the manga where I forgot to rebase the code. The method which I added to this file is "get_osd_stat". I will make the changes.

Reply:
I rebased the code and added the code. Check the https://github.com/red-hat-storage/cephci/pull/2340